cmd/register/examples.go

```diff
@@ -16,27 +18,41 @@ Registers all latest flytesnacks example
 
  bin/flytectl register examples  -d development  -p flytesnacks
 
+Registers specific release of flytesnacks example
```

Comment:
can we also add a link to github releases to flytectl from which users can get the version

Reply:
Done  
```
{"json":{"src":"examples.go:52"},"level":"info","msg":"Register started for flytesnacks v0.2.179 release https://github.com/flyteorg/flytesnacks/releases/tag/v0.2.179","ts":"2021-09-03T11:45:12+05:30"}

```